hudi-utilities/src/test/java/org/apache/hudi/utilities/transform/TestChainedTransformer.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.hudi.utilities.transform;
```

Comment:
Missing Apache Header.

Reply:
Good catch! @xushiyan , you can configure your IDE to add a file header template. For example, Intellij IDEA does support this feature. @leesf Can we let Travis check the license header?

Reply:
Guess rat check only works for source files.. we should fix that right away..

Reply:
Issue actually is the phase I think.. I locally tested that clean install /clean package will flag a file without header.. 
```
 <executions>
            <execution>
              <phase>package</phase>
              <goals>
                <goal>check</goal>
              </goals>
            </execution>
          </executions>
```

we need to change this to `verify` probably..

Reply:
Raised #1464